src/datastore/pg-write-store.ts

```diff
@@ -1772,6 +1772,9 @@ export class PgWriteStore extends PgStore {
     `;
     await this.refreshMaterializedView(sql, 'mempool_digest');
     const deletedTxs = deletedTxResults.map(r => r.tx_id);
+    for (const txId of deletedTxs) {
+      await this.notifier?.sendTx({ txId: txId });
```

Comment:
I think this is being called while in a sql transaction from the calling function, right? If so, seems like the deleted txs should be placed in a variable outside of the sql transaction function, then call the notifier after sql tx finishes?

Reply:
Good point, done